src/utils/cardHandler.js

```diff
@@ -0,0 +1,49 @@
+const SUITS = ["Hearts", "Diamonds", "Spades", "Clubs"];
+const VALUES = [
+  "A",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "10",
+  "J",
+  "Q",
+  "K",
+];
+
+function createCardDeck() {
```

Comment:
Você poderia escrever essa função usando `flatMap` e `map` ao invés de `for` loops.
https://devdocs.io/javascript/global_objects/array/flatmap